src/views/DApps.vue

```diff
@@ -15,7 +13,6 @@
         { label: 'Create dApps', path: 'create-dapps' },
```

Comment:
The page tabs seem to be cutting off on mobile view. Can you make this more responsive?
![bandicam 2021-03-25 13-38-45-185](https://user-images.githubusercontent.com/40356749/112419998-c15e4880-8d6f-11eb-9648-514af6855e7a.jpg)